.gitignore

```diff
@@ -27,3 +27,16 @@ website/static/static/katex.*
 website/static/static/fonts
 website/static/static/copy-tex.*
 website/static/static/mhchem.*
+
+# https://yarnpkg.com/advanced/qa#which-files-should-be-gitignored
+.yarn/*
+!.yarn/releases
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
```

Comment:
I don't see any files checked into for this director or `.website`, did you forget to add some files to this PR?

Reply:
@kevinbarabash They are files recommended by [the doc](https://yarnpkg.com/advanced/qa#which-files-should-be-gitignored). Yarn automatically uses Yarn in parent directories, so there are currently no files for `website`.